packages/core/src/docdb/wizards/createClusterWizard.ts

```diff
@@ -138,3 +157,41 @@ async function createEngineVersionPrompter(docdbClient: DocumentDBClient) {
         buttons: createCommonButtons(DocDBHelpUrl),
     })
 }
+
+async function createInstanceClassPrompter(docdbClient: DocumentDBClient, engineVersion: string) {
+    const options = await docdbClient.listInstanceClassOptions(engineVersion, DBStorageType.Standard)
+
+    const items: DataQuickPickItem<string>[] = options.map((option) => {
+        return {
+            data: option.DBInstanceClass,
+            label: option.DBInstanceClass ?? '(unknown)',
+            description: undefined,
+            detail: undefined,
+        }
+    })
+
+    if (items.length === 0) {
+        return new SkipPrompter('db.t3.medium')
+    }
+
+    return createQuickPick(items, {
+        step: 7,
+        title: localize('AWS.docdb.createInstance.instanceClass.prompt', 'Select instance class'),
+        buttons: createCommonButtons(DocDBHelpUrl),
+    })
+}
+
+function instanceCountItems(max: number = 16): DataQuickPickItem<number>[] {
+    const items = []
+
+    for (let index = 1; index <= max; index++) {
+        const item: DataQuickPickItem<number> = {
+            label: index.toString(),
+            data: index,
+        }
+        items.push(item)
+    }
+
+    items[2].description = '(recommended)'
```

Comment:
This seems arbitrary from my outside perspective. What makes the 3rd item special?

Perhaps add a comment to this code or find another way to make it more obvious what is happing.

Reply:
Also, if `max` is set to `< 3` you'll have issues. So you may want a guard as well.